sway/tree/arrange.c

```diff
@@ -0,0 +1,238 @@
+#define _POSIX_C_SOURCE 200809L
+#include <ctype.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+#include <wlr/types/wlr_output.h>
+#include <wlr/types/wlr_output_layout.h>
+#include "sway/debug.h"
+#include "sway/tree/arrange.h"
+#include "sway/tree/container.h"
+#include "sway/tree/layout.h"
+#include "sway/output.h"
+#include "sway/tree/workspace.h"
+#include "sway/tree/view.h"
+#include "list.h"
+#include "log.h"
+
+struct sway_container root_container;
+
+static void apply_horiz_layout(struct sway_container *parent);
+
+static void apply_vert_layout(struct sway_container *parent);
+
+void arrange_windows(struct sway_container *container) {
+	switch (container->type) {
+	case C_ROOT:
+		   arrange_root();
```

Comment:
You've still got messed up indentation here. There are an extra 3 spaces at the end.